crates/driver/openapi.yml

```diff
@@ -461,10 +490,23 @@ components:
     Quote:
       type: object
       properties:
-        sell_amount:
+        sellAmount:
+          $ref: "#/components/schemas/TokenAmount"
+        buyAmount:
           $ref: "#/components/schemas/TokenAmount"
-        buy_amount:
+        fee:
           $ref: "#/components/schemas/TokenAmount"
+    OrderQuote:
```

Comment:
Difference between Quote and OrderQuote is unclear and - if kept - needs to be documented.

Reply:
> if kept - needs to be documented.

I am still trying to find the best option here. Ideally, I am trying to avoid that, but due to mentioned limitations([1](https://github.com/cowprotocol/services/pull/2855#discussion_r1707496400), [2](https://github.com/cowprotocol/services/pull/2855#discussion_r1707132132)), maybe the best option is to keep it for a while. In any case, removing the PriceImprovement quote is going to be a breaking change. We can probably break it further by renaming back the OrderQuote to Quote.